core/src/impl/KokkosExp_IterateTileGPU.hpp

```diff
@@ -1200,12 +702,14 @@ struct DeviceIterateTile {
   KOKKOS_IMPL_DEVICE_FUNCTION DeviceIterateTile(
       const PolicyType& policy_, const Functor& f_, value_type_storage v_,
       const EmulateCUDADim3<index_type> gridDim_,
+      [[maybe_unused]] const EmulateCUDADim3<index_type> blockDim_,
```

Comment:
Any reason to  use `[[maybe_unused]]` here?
```suggestion
      const EmulateCUDADim3<index_type> blockDim_,
```